config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
 
+  get 'data_types/index'
```

Comment:
Do you need this route? `resources :data_types` should give you the index route too. This looks like it was autogenerated.